### ISSUE TITLE
Add options to recombine paragraphs

### ISSUE THREAD

**Issue by anuejn** [transcribee/transcribee#108]

**Comment by pajowu** [comment]
Duplicate of  #56?

**Comment by anuejn** [comment]
No, sorry for the unclear issue. This is more about having paragraphs that are longer than one sentence and are not "subtitle style" but rather "longform statement style"
